recipe/meta.yaml

```diff
@@ -25,6 +25,8 @@ requirements:
   run:
     - python
     - et_xmlfile
+    # Used to guard against quadratic blowup, etc.
+    - defusedxml
```

Comment:
Is there any downside in using it? I wonder why it wasn't added before.

Reply:
So it's been around since 2018.

ChatGPT: Downsides of using defusedxml with openpyxl
```
Using `defusedxml` with `openpyxl` does offer some security benefits by mitigating XML vulnerabilities. However, there are a few downsides to consider:

1. **Performance Overhead**: `defusedxml` adds an extra layer of processing to XML parsing, which could result in slightly slower performance compared to using the standard XML libraries.

2. **Compatibility Issues**: `defusedxml` aims to patch XML libraries to prevent security vulnerabilities, which could potentially lead to compatibility issues with certain XML files or libraries. While it's designed to maintain compatibility with most use cases, there might be edge cases where compatibility problems arise.

3. **Limited Functionality**: By design, `defusedxml` focuses on security rather than providing a full suite of XML processing capabilities. Therefore, you might encounter limitations or lack of support for certain XML features or functionalities.

4. **Dependency Management**: Introducing `defusedxml` as a dependency means managing an additional library in your project. This could potentially complicate dependency management and increase the complexity of your project's ecosystem.

5. **Learning Curve**: Developers who are unfamiliar with `defusedxml` may need to spend some time learning its usage and understanding its implications for their projects.

Despite these downsides, using `defusedxml` with `openpyxl` can significantly enhance the security of your application, especially when dealing with untrusted or potentially malicious XML files. It's important to weigh these drawbacks against the security benefits and make an informed decision based on your specific use case and requirements.
```

And most importantly, the PR where defusedxml was introduced to openpyxl chose to make it optional on purpose: https://foss.heptapod.net/openpyxl/openpyxl/-/merge_requests/300/
_Right, so a potentially significant performance penalty. No need to test all the combinations: it’s just a big file that gives the system a good workout. xlrd already has support for defusedxml but you have to pass it in as an option. It makes much more sense to encourage people to make defusedxml an optional dependency and use it if it’s around._

Reply:
Based on this I would remove this dependency.